managed/utils/interceptors/interceptors.go

```diff
@@ -89,10 +89,12 @@ func Unary(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, han
 	l := logrus.WithField("request", logger.MakeRequestID())
 	ctx = logger.SetEntry(ctx, l)
 
+	ctx = SetCallerOrigin(ctx, info.FullMethod)
```

Comment:
BTW can we have it inside extension itself? Why we need to check origin here?

Reply:
Inside extension we have not access to info.FullMethod

Reply:
We can, but in this case we should extends from:
```golang
type ServerExtension interface {
	// ...
	ServerStreamMsgReceivedCounterValues(ctx context.Context) []string
	// ...

}
```
to:
```golang
type ServerExtension interface {
	// ...
	ServerStreamMsgReceivedCounterValues(ctx context.Context, serviceName, methodName string) []string
	// ...

}
```
Set to context it's easy way

Reply:
If we consider this extension approach as a general way to add some custom functionality then in totally makes sense to be able to get all formation that other interceptors has. Otherwise it's not enough just to add extension and you need to prepare and put data to the context in some other interceptor.

It looks like that `ServerExtension` should follow gRPC interceptor signatures (in parameters), no?

Reply:
`go-grpc-prometheus` lib provides metrics interceptors, why we don't extend these interceptors with our custom logic, but instead putting it in some other interceptor? Can we make something like `grpc_prometheus.UnaryServerInterceptor(extension.Extension("lablel", funcThatCountsWhatWeNeed()))`